src/video-grid/VideoTileSettingsModal.module.css

```diff
@@ -0,0 +1,76 @@
+.content {
+  margin: 27px 34px;
+}
+
+.localVolumePercentage {
+  width: 40px;
```

Comment:
I would suggest specifying this in `ch` units so it scales as expected no matter the font or font size